examples/ecies-example.cpp

```diff
@@ -0,0 +1,148 @@
+/*
+ * #%L
+ * %%
+ * Copyright (C) 2020 BMW Car IT GmbH
+ * %%
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ * #L%
+ */
+
+#include <mococrw/ecies.h>
+#include <mococrw/hash.h>
+#include <mococrw/util.h>
+#include <mococrw/openssl_wrap.h>
+
+#include <iostream>
+
+using namespace mococrw;
+
+struct EciesEncryptResult {
+    std::vector<uint8_t> ephKey;
```

Comment:
```suggestion
    std::vector<uint8_t> ephemeralKey;
```
please be verbose with names in the examples. People don't have a VT-100 anymore and read the examples to understand what is going on…